### PR TITLE
added signicat authorization scheme

### PR DESCRIPTION
**Pull request by michielmayer**

### CODE REVIEW

File: signicat/Issues/authorization/description.xml [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+			</Description>
+		</Attribute>
+		<Attribute id="referenceMoment">
+			<Name>
+				<en>Reference moment</en>
+				<nl>Peilmoment (opvraagmoment uit handelsregister)</nl>
+			</Name>
+			<Description>
+				<en>Reference moment is the moment the data is extracted from the trade register (format yyyymmddhhmmssmmm)</en>
+				<nl>Peilmoment is het moment dat de gegevens uit het handelsregister zijn onttrokken (formaat jjjjmmddhhmmssmmm)</nl>
```
Comment (sanderhollaar) [inline signicat/Issues/authorization/description.xml:109]: s/ontrokken/opgevraagd/

File: signicat/Issues/authorization/description.xml [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+			</Name>
+			<Description>
+				<en>Start date of the organization's registration</en>
+				<nl>Aanvangsdatum van de registratie van de maatschappelijke activiteit</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="referenceMoment">
+			<Name>
+				<en>Reference moment</en>
+				<nl>Peilmoment (opvraagmoment uit handelsregister)</nl>
```
Comment (sanderhollaar) [inline signicat/Issues/authorization/description.xml:105]: ~~(opvraagmoment uit handelsregister)~~

File: signicat/Issues/authorization/description.xml [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+	<ShortName>
+		<en>Demo authorization</en>
+		<nl>Demo machtiging</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>signicat</IssuerID>
+	<CredentialID>authorization</CredentialID>
+	<Description>
+		<en>Authorization for an organization, issued by an official registered in the Chamber of Commerce trade register.</en>
+		<nl>Machtiging voor een organisatie, uitgegeven door een functionaris die geregistreerd staat in het KVK handelsregister.</nl>
```
Comment (sanderhollaar) [inline signicat/Issues/authorization/description.xml:15]: maybe remove the trailing dots to match the other Description fields

File: signicat/Issues/authorization/description.xml [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+				<nl>Rechtsvorm</nl>
+			</Name>
+			<Description>
+				<en>Legal entity of the owner of the organisation</en>
+				<nl>Rechtsvorm van de eigenaar van de maatschappelijke activiteit</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="address">
+			<Name>
+				<en>Address of the organisation</en>
```
Comment (sanderhollaar) [inline signicat/Issues/authorization/description.xml:74]: ~~of the organisation~~

File: signicat/Issues/authorization/description.xml [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+				<nl>Adres van de organisatie</nl>
```
Comment (sanderhollaar) [inline signicat/Issues/authorization/description.xml:75]: ~~van de organisatie~~

**Review by sanderhollaar — CHANGES_REQUESTED** [review]

**Comment by michielmayer** [comment]
In the last commit, I changed the order of the attributes. We decided It is better to have the company and the authorization on top, so the user does not have to scroll in the app to decide if he shares the correct credential.

**Review by sanderhollaar — DISMISSED** [review]

**Review by sanderhollaar — APPROVED** [review]

**Review by bobhageman — APPROVED** [review]

**Review by ivard — APPROVED** [review]

**Pull request merged**